packages/design-system/src/components/TextField/TextField.jsx

```diff
@@ -8,6 +8,9 @@ import uniqueId from 'lodash.uniqueid';
 export { unmaskValue } from './Mask';
 
 export class TextField extends React.PureComponent {
+  // Set component name to make child.type.displayName available to other components (eg. Autocomplete)
+  static displayName = 'TextField';
```

Comment:
why is this one static?

Reply:
Using `static` to set the `displayName` property for the class.
Ah, just realized I should try declaring class static property after the class definition, like what I did for the functional component. Then wont need the `@babel/plugin-proposal-class-properties` and `babel-eslint`

Reply:
Will give that a try

Reply:
Have updated to declare class property `displayName` after the class definition and removed the `babel` dependencies